doc/classes/Control.xml

```diff
@@ -715,6 +715,20 @@
 				Give up the focus. No other control will be able to receive keyboard input.
 			</description>
 		</method>
+        <method name="find_next_valid_focus" qualifiers="const">
```

Comment:
You have spaces in this snippet and the file is formatted with tabs. Also, please put the new methods into the doc in the alphabetical order.

Reply:
Documentation fixed.